packages/cid-verifier/src/bindings.d.ts

```diff
@@ -9,6 +9,7 @@ export interface EnvInput {
   GOOGLE_EVALUATE_SAFE_CONFIDENCE_LEVELS: Array<string>;
   GOOGLE_CLOUD_API_URL: string;
   GOOGLE_CLOUD_API_KEY: string;
+  IPFS_WWW_TLD: string;
```

Comment:
```suggestion
  IPFS_GATEWAY_TLD: string;
```

let's call it gateway